### ISSUE TITLE
resizeable device doesn't resize

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Perundurai-Products/microemu#20]
```
What steps will reproduce the problem?
1. select the resizeable device from the device list.
2. open a midlet.
3. change the emulator size (either with mouse or using the Resize button.

What is the expected output? What do you see instead?
The display area should be resized appropriately.
Instead, the window area is resized, but the midlet's display area remains
the same, i.e. the midel display takes up only the same small portion of
the window and the rest is just blank white space. Also, the sizeChanged
event is fired with the previous dimensions, not the new ones.

What version of the product are you using? On what operating system?
2.0.3 on Linux (Sun JVM 1.6.0_16).

Please provide any additional information below.


```

Original issue reported on code.google.com by `amichair` on 2 Jan 2010 at 7:16

**Comment by GoogleCodeExporter** [comment]
```
clarification: the problem occurs in non-fullscreen mode. If 
setFullScreenMode(true)
is used, the resizing works properly.
```

Original comment by `amichair` on 2 Jan 2010 at 7:22

**Comment by GoogleCodeExporter** [comment]
```
The patch fixes this issue. It's not perfect, but works well (and certainly 
better
than before, where it didn't work at all).
```

Original comment by `amichair` on 3 Jan 2010 at 12:00

Attachments:
- [fix_resize.diff](https://storage.googleapis.com/google-code-attachments/microemu/issue-20/comment-2/fix_resize.diff)

**Comment by GoogleCodeExporter** [comment]
```
An improved patch, excluding small unrelated change, and fixing the initial 
display
size (before resizing, using previously saved settings).
```

Original comment by `amichair` on 3 Jan 2010 at 3:13

Attachments:
- [fix_resize.diff](https://storage.googleapis.com/google-code-attachments/microemu/issue-20/comment-3/fix_resize.diff)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in trunk (future version 3.0)
```

Original comment by `bar...@gmail.com` on 6 Jan 2010 at 11:18
- Changed state: **Fixed**
- Added labels: **Milestone-3.0**
